docs/contact.md

```diff
@@ -2,8 +2,9 @@
 
 For more information:
 
-* visit our website [https://www.eessi-hpc.org](https://www.eessi-hpc.org)
+* visit our website [https://www.eessi.io](https://www.eessi.io)
 * consult our documentation at [https://eessi.github.io](https://eessi.github.io/docs)
+* ask for help at our [support portal](https://gitlab.com/eessi/support)
```

Comment:
also here, I would point to the support section of our docs, which provides more info:

```suggestion
ask for help at our [support portal](support.md)
```

Reply:
Agreed, but on this page I left it as a bullet point, let me know if you'd prefer otherwise